demos/http/http_demo_plaintext/http_demo_plaintext.c

```diff
@@ -456,34 +427,37 @@ static int sendHttpRequest( HTTPTransportInterface_t * pTransportInterface,
     }
     else
     {
-        LogError( ( "Sending HTTP %s request to %s%s failed with status = %u.",
-                    pMethod, SERVER_HOST, pPath, httpStatus ) );
+        LogError( ( "Failed to send HTTP %s request to %s%s: Error=%s.",
+                    pMethod, SERVER_HOST, pPath, HTTPClient_strerror( httpStatus ) ) );
     }
 
-    if( httpStatus == HTTP_SUCCESS )
-    {
-        return EXIT_SUCCESS;
-    }
-    else
+    if( httpStatus != HTTP_SUCCESS )
     {
-        return EXIT_FAILURE;
+        returnStatus = EXIT_FAILURE;
     }
+
+    return returnStatus;
 }
 
 /*-----------------------------------------------------------*/
 
 /**
  * @brief Entry point of demo.
+ *
+ * This example resolves a domain, then establishes a plaintext TCP connection to
+ * an HTTP server. After which, HTTP_Client APIs are used to send a GET, HEAD,
+ * PUT, and POST request in that order. For each request, an HTTP response from
```

Comment:
(Nit) "For each request, ~~an~~ the HTTP response received from the server is logged."